stories/MenuSelect.stories.js

```diff
@@ -105,4 +105,25 @@ stories
       displayName,
       filterProps,
     }
+  )
+  .addWithJSX(
+    'with localized count',
+    () => (
+      <WrapWithHits linkedStoryGroup="MenuSelect">
+        <MenuSelect
+          attributeName="category"
+          defaultRefinement={text('defaultSelectedItem', 'Bathroom')}
+          transformItems={items =>
+            items.map(item => {
+              item.count = (item.count + 1000).toLocaleString();
```

Comment:
Avoid to modify an object inside a map, try to always return a new object. Prefer something like:

```js
items.map(({ count, ...item }) => ({
  ...item,
  count: count.toLocaleString()
}));
```

Reply:
Will update the PR. Thanks.